config/default.yml

```diff
@@ -3089,7 +3089,19 @@ Style/ArgumentsForwarding:
   Enabled: pending
   AllowOnlyRestArgument: true
   UseAnonymousForwarding: true
+  ForbiddenRestArgumentNames:
```

Comment:
I think "Redundant" is probably a better word than "Forbidden" for the purpose of this cop.